### ISSUE TITLE
Widgets not obeying scroll_fixed_width

### ISSUE THREAD

**Issue by lyra-scarlet** [qtile/qtile#4331]
### The issue:

Qtile version: v0.22.1 (latest)

Seemingly the `scroll_fixed_width` parameter of widgets is not correctly obeyed. Tested using default config and setting `screens` as follows:
```python
screens = [
    Screen(
        top=bar.Bar(
            [
                widget.WindowName(
                    width=1000,
                    scroll=True,
                    scroll_fixed_width=True,
                    background = "#ff00ff"
                ),
            ],
            24,
        ),
    ),
]
```
Expected to see a magenta bar up to 1000px but only extends as far as the window name text. 

This behaviour appears to apply to other widgets too. As an example, the Mpris2 widget which scrolls by default:
```python
widget.Mpris2(
    background = "#ff00ff",
    width=1000,
    scroll_fixed_width=True,
),
```

Does not produce any log information.

### Required:

- [X] I have searched past issues to see if this bug has already been reported.

**Comment by elParaguayo** [comment]
I think this was fixed after v0.22.1.  See: https://github.com/qtile/qtile/commit/b6061282c895e61973109e5da368870ff1878c86

Works on latest git for me.

**Comment by lyra-scarlet** [comment]
Saw that the issue was closed, did not realise that the relevant update had not yet rolled out... thank you very much!
